one_shot_scripts/validate_colors.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Cube = require('../models/cube');
+const config = require('../config/database');
+
```

Comment:
This script will OOM on live server. Can't load all cubes into memory at once.

Reply:
Will fix